apps/web/src/components/tx/confirmation-views/ManageSigners/get-new-safe-setup.ts

```diff
@@ -11,15 +12,17 @@ export function getNewSafeSetup({
   txInfo,
   txData,
   safe,
+  signerNames = {},
 }: {
   txInfo: TransactionInfo
   txData: TransactionDetails['txData']
   safe: ExtendedSafeInfo
+  signerNames?: Record<string, string>
 }): {
-  newOwners: Array<string>
+  newOwners: Array<AddressInfo>
   newThreshold: number
 } {
-  let newOwners = safe.owners.map((owner) => owner.value)
+  let ownerAddresses = safe.owners.map((owner) => owner.value)
```

Comment:
I think we should normalize here as well, just in case.
```suggestion
  let ownerAddresses = safe.owners.map((owner) => checksumAddress(owner.value))
```